playwright/pages/SliceBuilderPage.ts

```diff
@@ -1,11 +1,19 @@
 import { expect, Locator, Page } from "@playwright/test";
 
+import { AddNewVariationDialog } from "./components/AddNewVariationDialog";
```

Comment:
❓Why not `AddVariationDialog` just a bit shorter since it's for sure `New`?
I don't really have a preference, more a question here 😉